### DISCUSSION TITLE
part-ii/2009-100/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#298] [Question comments]
Part II 2009 Number Theory: Paper 2, Section $\mathbf{I}$, $1 \mathrm{G}$ 

https://questions.tripos.org/part-ii/2009-100/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

The first part is bookwork.

For the computation of the Jacobi symbol, we have

$$
\begin{align*}
(\frac{261}{317}) &= (\frac{317}{261}) \\
&= (\frac{56}{261}) \\
&= (\frac{14}{261}) \\
&= (\frac{2}{261})(\frac{7}{261}) \\
&= -(\frac{7}{261}) \\
&= -(\frac{261}{7}) \\
&= -(\frac{2}{7}) \\
&= -1 \\
\end{align*}
$$

</details>
